schema.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$comment": "Abstract definition of a collection of Jobs/CronJobs. Each generated Job/CronJob must include the fields `'name`, `image`, `namespace`, and `schedule`. When a python implementation of >v4 of the json schema draft is available use of`allOf` should be replaced with `merge`.",
+  "allOf": [
+    {
+      "$ref": "#/definitions/job"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "jobs": {
+          "type": "array",
+          "comment": "An array of jobs that will inherit, and can override, properties from the top level and `namespaceOverrides`.",
+          "items": {
+            "$ref": "#/definitions/job"
+          }
+        },
+        "namespaces": {
+          "type": "array",
+          "comment": "An array of namespaces for which to generate jobs. Each namespace here will result in the generation of a job that is identical in all aspects but namespace",
+          "items": {
+            "type": "string"
+          }
+        },
+        "namespaceOverrides": {
+          "type": "object",
+          "comment": "An object mapping namespaces to a set of properties given a generated job belongs to the specified namespace.",
+          "patternProperties": {
```

Comment:
why was patternProperties necessary if the pattern is empty?

Reply:
I don't think there's another way to specify the types of an objects values when the key can be any string

Reply:
would be nice to make sure

Reply:
looks like `additionalProperties` is the way to go